app/assets/javascripts/wizard/addon/components/homepage-preview.js

```diff
@@ -370,12 +370,9 @@ export default createPreviewComponent(659, 320, {
 
     const drawLine = (y) => {
       ctx.beginPath();
-      ctx.strokeStyle = darkLightDiff(
-        colors.primary,
-        colors.secondary,
-        90,
-        -75
-      );
+      ctx.strokeStyle = colors.primary_low // accounts for hard-set color variables in solarized themes
+        ? colors.primary_low
+        : darkLightDiff(colors.primary, colors.secondary, 90, -75);
```

Comment:
```suggestion
      // accounts for hard-set color variables in solarized themes
      ctx.strokeStyle = colors.primary_low || darkLightDiff(colors.primary, colors.secondary, 90, -75);
```

(you may need to run prettier)